docs/book/src/providers.md

```diff
@@ -22,7 +22,7 @@ This document highlights the implementation steps for adding a secrets-store-csi
 
 ### Implementation details
 
-The driver as of `v0.0.14` adds an option to use gRPC to communicate with the provider. This is an alpha feature and is introduced with a feature flag `--grpc-supported-providers`. The `--grpc-supported-providers` is a `;` delimited list of all providers that support gRPC for communication. The driver will communicate with the provider using gRPC only if the provider name is in the list of supported providers in `--grpc-supported-providers`.
+The driver as of `v0.0.14` adds an option to use gRPC to communicate with the provider. This is an alpha feature and is introduced with a feature flag `--grpc-supported-providers`. The `--grpc-supported-providers` is a `;` delimited list of all providers that support gRPC for communication. This flag is no longer necessary as this is the only supported communication mechanism.
```

Comment:
We should add a note about it being only supported mechanism as of v0.0.20 release of the driver

Reply:
Updated the note to say `v0.0.21` since the flag is still necessary in `v0.0.20` to know which providers to connect to.